<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmpFrame</class>
 <widget class="QWidget" name="AmpFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WahWahWidget {color: transparent;}</string>
  </property>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Standard</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="maxPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompInstance">
    <string>Amplifier</string>
   </property>
   <property name="stompName">
    <string>Author:</string>
   </property>
   <property name="value1Title">
    <string>Definition</string>
   </property>
   <property name="value2Title">
    <string>Power Sagging</string>
   </property>
   <property name="value3Title">
    <string>Pick</string>
   </property>
   <property name="value4Title">
    <string>Compressor</string>
   </property>
   <property name="value5Title">
    <string>Clarity</string>
   </property>
   <property name="value6Title">
    <string>Tube Shape</string>
   </property>
   <property name="value7Title">
    <string>Tube Bias</string>
   </property>
   <property name="value8Title">
    <string>Direct Mix</string>
   </property>
   <property name="value9Title">
    <string/>
   </property>
   <property name="value10Title">
    <string/>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string/>
   </property>
   <property name="value13Title">
    <string/>
   </property>
   <property name="value14Title">
    <string/>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string/>
   </property>
   <property name="value17Title">
    <string/>
   </property>
   <property name="value18Title">
    <string/>
   </property>
   <property name="value19Title">
    <string/>
   </property>
   <property name="value20Title">
    <string/>
   </property>
   <property name="value21Title">
    <string/>
   </property>
   <property name="value22Title">
    <string/>
   </property>
   <property name="value23Title">
    <string/>
   </property>
   <property name="value24Title">
    <string/>
   </property>
   <property name="eqBand1Value">
    <double>0.000000000000000</double>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
  <widget class="QToasterDial" name="tubeBiasDial">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="tubeShapeDial">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
  </widget>
  <widget class="QToasterDial" name="clarityDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
  </widget>
  <widget class="QToasterDial" name="directMixDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="definitionDial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="powerSaggingDial">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="pickDial">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Bi</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
  </widget>
  <widget class="QToasterDial" name="compressorDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="precision">
    <UInt>1</UInt>
   </property>
   <property name="unit">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>definitionDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powerSaggingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pickDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>compressorDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tubeShapeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue6(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tubeBiasDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue7(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directMixDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue8(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clarityDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
